students/tlugosan/lesson03/mailroom.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+""" Automate a script to send a thank you mail, print a report of donor or quit the program."""
+
+table_dictionary = {
+    'Toni Orlando': [150.00, 200.00, 100.00],
+    'Amanda Clark': [1800.00],
+    'Robin Hood': [1234.56, 4500.34, 765.28],
+    'Gina Travis': [523.10, 75.00],
+    'Mark Johnson': [850.00, 20.14]
+}
```

Comment:
for the first revision of mailroom this should have been a list (to practice list functionality)... but yes dictionary makes more sense here and will be used in lesson 4.